driverlessai_experiments/iid/credit_card_experiment/credit_card_default.ipynb

```diff
@@ -231,13 +234,13 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 12,
+   "execution_count": 7,
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "{'dataset': {'key': 'ee30bf26-c45f-11e9-b1a0-0242ac110002',\n",
+       "{'dataset': {'key': '72e4a2cc-024b-11ea-b1a2-0242ac110002',\n",
```

Comment:
Is this hardcoded key working? Because DAI always assigns unique key..or is it just example of code execution?

Reply:
I ran that ipynb as is, and everything worked. So i'm guessing that it's an example of one that DAI actually assigned.

Reply:
This key is being printed by DAI - it will be different for other people but will run fine as it's just output